### ISSUE TITLE
Ensure that a11y development hasn't broken stable PhET-iO API.

### ISSUE THREAD

**Issue by zepumph** [phetsims/faradays-law#115]
I noticed that the WASD queues are shown when a11y not enabled while working on phet-io for Faradays Law today. @samreid, @chrisklus and I are getting very close to publishing Faradays law for phet-io, so this should be tackled at a reasonably high priority I think.

It brings up a good topic for further conversation too though. @mbarlow12 what is the best way to proceed? How much work do you think is necessary in regression testing the sim on master right now for a phet-io release?

**Comment by mbarlow12** [comment]
@zepumph thanks for the heads up. I added a check for the `accessibility` query parameter that adds the letter keys only if enabled. Let me know if this doesn't resolve the issue.

**Comment by mbarlow12** [comment]
Updated the cue to disappear when starting mouse/keyboard drag. Is there anything else you've noticed, @zepumph, that needs to be addressed prior to the next release (or before closing)?

**Comment by zepumph** [comment]
I'm not totally sure what is left for this issue. I think someone will need to go through the sim before we take it to RC and give it a once over to make sure that the code seems production ready.

**Comment by mbarlow12** [comment]
@zepumph I'm happy to take the lead on that since I'll be digging into this pretty frequently. @samreid had mentioned having a meeting to discuss some of the issues that have arisen, but that seems like it's not happening?

**Comment by zepumph** [comment]
Yeah. @samreid please chime in on potential issues that you would want to discuss, but my main concern is prototype code when we move to dev and RC. It would be good to have that cleaned up, and then we can take another phet-io pass to spot check the phet-io api.

**Comment by samreid** [comment]
> @samreid had mentioned having a meeting to discuss some of the issues that have arisen, but that seems like it's not happening?

We've been postponing until we are closer to Faraday's Law RC.  The main thing will be to make sure we are at a good stable point for sim SHAs, not catching changes partway through.

**Comment by samreid** [comment]
Unassigning self for now, will check in again when we are closer to RC.

**Comment by jessegreenberg** [comment]
> The main thing will be to make sure we are at a good stable point for sim SHAs, not catching changes partway through.

Do we also need to make sure that a11y is solid enough to reduce the possibility it of making an API breaking change? If so, we may want to work on this before the RC is imminent.

**Comment by zepumph** [comment]
>Do we also need to make sure that a11y is solid enough to reduce the possibility it of making an API breaking change? If so, we may want to work on this before the RC is imminent.

Can you explain your thoughts a bit more. If you are talking about a11y features and components being instrumented (like the intersection of a11y and phet-io), then I would think not right now. But if instead you are talking about the a11y code (for example describers and the new arrow queues) being stable enough such than they will not likely move from the magnet to the screenView or something, therefore breaking the phet-io API if they are instrumented, then yes that seems like something to be hammered out.

**Comment by jessegreenberg** [comment]
>  But if instead you are talking about the a11y code (for example describers and the new arrow queues)

Right, that is what I was wondering about. We may need to add Properties/Emitters, or move/refactor code, or hopefully generalize and move out of the sim entirely. But sounds like this is probably not a problem if we avoid instrumenting sim-specific a11y code.

**Comment by chrisklus** [comment]
@zepumph and I are thinking that more work needs to be done before this issue can be addressed.

**Comment by zepumph** [comment]
This issue is out of date, and more thorough review will be needed before publishing FL, let's pick things up in #158.
